### ISSUE TITLE
AWS Amplify: Domain Activation Failed

### ISSUE THREAD

**Issue by reyts** [aws-amplify/amplify-console#2191]
### Before opening, please confirm:

- [X] I have checked the [custom domains troubleshooting guide](https://docs.aws.amazon.com/amplify/latest/userguide/custom-domain-troubleshoot-guide.html).
- [X] I have read the guide for [submitting bug reports](https://github.com/aws-amplify/amplify-console/blob/master/CONTRIBUTING.md).
- [X] I have done my best to include a minimal, self-contained set of instructions for consistently reproducing the issue.

### App Id

d26fp7zuh71ka8

### Region

us-east-1

### Step

Domain Activation

### Status

Failed

### Issue or question

I have been using AWS Amplify for sometime now. I got word from a user that the site was down and I have tried several options to activate the site to no avail. I have used the documentation provided to troubleshoot and still did not work. I don't have any cloudfront setup talkless of having a cloudfront distribution ID.

The domain is hosted with Route53 because I wanted to avoid godaddy issues.

I have already looked at these documents

https://docs.aws.amazon.com/amplify/latest/userguide/custom-domain-troubleshoot-guide.html
https://docs.aws.amazon.com/amplify/latest/userguide/custom-domain-troubleshoot-guide.html#i-get-a-cnamealreadyexistsexception-error

Kindly assist

### Error message

Something went wrong. Please refer to our troubleshooting guide below to diagnose the issue. After fixing the issue, choose retry below.
Error message: One or more of the CNAMEs you provided are already associated with a different resource.

### Additional information

![image](https://user-images.githubusercontent.com/73900693/130339992-fac05e5f-92b6-4493-93c5-8cb59110d401.png)

**Comment by reyts** [comment]
Can anyone help with this ticket?

**Comment by reyts** [comment]
I am going to go ahead to close this ticket I found a temporary workaround.

I went to AWS Amplify to add one of the spare domains I had purchased on GoDaddy.
Followed the instructions [here](https://docs.aws.amazon.com/amplify/latest/userguide/to-add-a-custom-domain-managed-by-godaddy.html) and as prompted on AWS amplify. 

Then went to Route 53 to redirect/forward from route 53 hosted domain to the Godaddy hosted domain using the instructions[here](https://aws.amazon.com/premiumsupport/knowledge-center/route-53-redirect-to-another-domain/)
